test/process/process_records_e2e.js

```diff
@@ -17,17 +18,40 @@ const testEventProps = {
 test('process local query indexes and relationships', async (t) => {
   const alice = await buildPlayer(['observation'])
   try {
-    const { cells: [observation] } = alice
+    const { cells: [observation], graphQL } = alice
 
     // SCENARIO: write records
     const process = {
       name: 'test process for linking logic',
     }
+    const process2 = {
+      name: 'test process2',
+    }
     const pResp = await observation.call('process', 'create_process', { process })
+    const p2Resp = await observation.call('process', 'create_process', { process: process2 })
     console.log(pResp)
```

Comment:
```suggestion
```

can you take out all the other console.log statements from the other ones as well?